# This message is a container for a Turtlebot's MMA-generated name, and its Decawave position.
string name
geometry_msgs/PoseStamped pose